technology_and_tooling/testing/automated_testing.md

```diff
@@ -74,13 +74,13 @@ $ cd inflammation
 If on WSL or Linux (e.g. Ubuntu or the Ubuntu VM), then do:
 
 ~~~bash
-$ wget https://www.uhpc-training.co.uk/material/software_architecture_and_design/procedural/inflammation/inflammation.zip
+$ wget https://www.uhpc-training.co.uk/material/HPCu/software_architecture_and_design/procedural/inflammation/inflammation.zip
```

Comment:
```suggestion
$ wget https://train.oxrse.uk/HPCu/software_architecture_and_design/procedural/inflammation/inflammation.zip
```

Reply:
Thanks @martinjrobins I've updated the links now :)